src/components/dropdowns/_dropdowns.scss

```diff
@@ -0,0 +1,93 @@
+// your components variables
+$dropdownFontSize: 12px;
+$dropdownHeight: 32px;
+$dropdownElementHeight: 36px;
+$dropdownMargin: 10px;
+
+$include-html: false !default;
+
+@if ($include-html) {
+
+  .mint-dropdown {
+    @include component;
+    @extend .mint-icon-arrow_down;
+
+    color: $grayAlt;
+    background-color: $grayAltLight;
+    border: 2px solid $grayAltLight;
+    border-radius: 16px;
+    font-size: $dropdownFontSize;
+    line-height: 1;
+    height: $dropdownHeight;
+    text-transform: uppercase;
+    font-family: $fontFamilySecondary;
+    cursor: pointer;
+
+    &--full {
+      width: 100%;
+    }
+
+    &:before {
+      $iconSize: $dropdownFontSize + $iconBoostValue;
+      font-size: $iconSize;
+      font-weight: bold;
+      position: absolute;
+      top: $dropdownHeight/2 - 1;
+      right: $dropdownHeight / 2 - $iconSize / 2;
+      pointer-events: none;
```

Comment:
it's for arrow, it's almost the copy of arrow from select,
maybe i could prepare one mixin for both?